starttls-policy/starttls_policy/configure.py

```diff
@@ -49,6 +51,11 @@ def manual_instructions(self):
                     mta_name=self.mta_name,
                     instructions=self._instruct_string()))
 
+    def expired_warning(self):
+        """Warns user about policy list expiration."""
+        six.print_("\nACTION REQUIRED: policy list expired. Probably, policy update ceased. "
+                   "Generating empty config.\n", file=sys.stderr)
```

Comment:
Let's change this text to something like:
"\nACTION REQUIRED: your policy list at {config_location} has expired! Generating empty config.\n
Check to see whether your update mechanism (cronjob, systemd timer) is working.\n"